src/check.ts

```diff
@@ -117,17 +119,21 @@ document.addEventListener('DOMContentLoaded', () => {
     async function (e) {
       e.preventDefault();
 
+      const browserInfo = (await getBrowserInfoTask(iopayportalClient).run()).getOrElse({
+        ip: '',
```

Comment:
ok for the moment if it doesn't work, we go ahead anyway. We should understood later if these info in our case are actually mandatory and used. @GiovanniMancini @infantesimone